src/libpriv/rpmostree-postprocess.c

```diff
@@ -1893,8 +1908,15 @@ rpmostree_commit (int            rootfs_fd,
         return FALSE;
     }
 
-  if (!ostree_repo_prepare_transaction (repo, NULL, cancellable, error))
-    return FALSE;
+  /* See comment above */
+  const gboolean use_txn = getenv ("RPMOSTREE_COMMIT_NO_TXN") == NULL &&
```

Comment:
Minor: can we wrap in parentheses here and/or put the whole boolean expression on their own line?